Makefile

```diff
@@ -67,8 +67,23 @@ local:
 	@rm -rf /var/tmp/$(PKGNAME)-$(VERSION)
 	@echo "The archive is in $(PKGNAME)-$(VERSION).tar.gz"
 
+test-container:
+	sudo docker build -t larskarlitski/lorax-tests:latest -f Dockerfile.test .
+
+check-in-docker:
+	sudo docker run --rm -ti \
+          -v `pwd`:/lorax:ro \
+          -e top_builddir=/build \
+          -e top_srcdir=/lorax \
```

Comment:
`tests/testenv.sh` is supposed to configure these variables. Why do you have to pass them by hand ?